### ISSUE TITLE
Best lang app for laravel

### ISSUE THREAD

**Issue by nam-co** [christofferok/langly#2]
Hi Cristoffer, thanks for the app, here are some ideas/suggestions/questions

1. is there a way to change the text directly in the view file, from langly? (for the first lang)
Example: if I put 'hope and DEliverance' but I decide to go to the view file (edit first lang) then reload on langly and retype a new row in the second lang, it would be cool to edit directly in langly and have the app change all the files in the app
But I guess is just better to al have a lang file for the primary language.

2. When "not in app", is there a way to erase it from the file.json?
Maybe a checkbox, label: Include on save.

3. Do you think is a good idea to order the translations in alphabetical order in the file.json save?

4. is it possible with electron to add a update langly, when new versions come out?

5. is it possible to allow mac right click to see the correct spelling in the input fields? right now it doesn't allow paste

6. Could it be possible in the lang file dropdown, a "All" option, that shows all translations, example in attach image
![langly-example](https://cloud.githubusercontent.com/assets/3951529/26162954/ca346d82-3aed-11e7-9f18-762bff6cb313.png)

Thanks for sharing this app and your time
You should sell it for at least 10usd
Regards
Michael

**Comment by christofferok** [comment]
Hi Michael,

Thanks for the feedback. 

1. Not possible at this time, but that's a very interesting idea. Especially if you use the same key in multiple places. 

2. I've been looking into a good way to implement this. Will try to push something out within a few weeks. 

3. Yes. Sorted on key would be better. I'll look into it. 

4. Working packaging the app for all platforms and have it auto update. Coming soon 😄 

5. Did a quick test with a few spelling packages and had some issues with compiling, but will look into it at a later time. Copy/paste I'll look into right away though 👍 

6. Love the idea. Just don't have the capacity to tackle it right now.

Pull requests are of course welcome 🙏 

**To summarize I'll try to get these things addressed in the next version:**
- Better packaged for all platforms
- Auto update
- Copy/paste
- Sort keys on save

**Comment by nam-co** [comment]
Hi Cristoffer, thanks for the response, I wishhh I could pull request and code, but Im not any good in vue or js in general, I tried looking in No. 2 but Im afraid Im going to mess it up, but looking arround  I found some links, most probably you have seen them, but here they are anyway, thanks for your time, great app

https://github.com/electron/electron/issues/4068
https://pracucci.com/atom-electron-enable-copy-and-paste.html
https://www.npmjs.com/package/electron-spell-check-provider

**Comment by christofferok** [comment]
Just pushed out a new version that addresses most of your suggestions + some extra stuff 😄

**Comment by nam-co** [comment]
👍 thanks Christoffer

**Comment by nam-co** [comment]
Hi Christoffer, great version, thank you very much, I send you a new icon idea, don’t know if you saw it?

Thanks for sharing C

Regards
Michael Mc.

**Comment by nam-co** [comment]
Hi Christoffer, hope everything is good, I was wondering are you still working on this? I hope, because is the easiest 

Christoffer I notice 3 things I hope you'll consider:
1. is there a way to show doubles (same key name)? so we can eliminate one
2. Option to eliminate key in all languages at once (right now you have to eliminate for each lang file)
3. I wish Langly would search the route files, I tried a __('') in the console.php but it couldn't find it

Regards
Michael

**Comment by christofferok** [comment]
Hi Michael,

The project is still active, but I haven't had the need for any new features so there has not been any updates. 

1. Do you mean that the `<language>.json` has two identical keys? When loaded it will just show the last value for that key and when you save it will only contain that 1 key. 
2. Might look into this at a later date. 
3. I've pushed up new code where it will also search the `routes` directory. To use it pull down the code and run `npm install` and `npm start`. Will probably not have time to release new binaries at the moment.

**Comment by nam-co** [comment]
Hi Christoffer, thanks for the update , I will double check again point 1 to see if it still happens and under witch conditions, good luck, pls remember the binary when you can

**Comment by christofferok** [comment]
I've published new binaries for you 👍🏼
